env.io-backend

```diff
@@ -121,7 +121,7 @@ THIRD_PARTY_CONFIG_LIST=""
 # ------------------------------------
 # APPBACKENDLI 
 # ------------------------------------
-IS_APPBACKENDLI=false
+IS_APPBACKENDLI=true
```

Comment:
Interesting. This was the reason because the `/notify` endpoint and `/lock` session endpoints returns `404`. Documentation over this variable is not sufficient in here and into `io-backend`. We should remember to update the README in `io-backend` to explicit more this functionality behavior.
cc. @gquadrati